### DISCUSSION TITLE
XML-Import: Objectclass Title

### DISCUSSION THREAD

**Discussion by MLindenhofer** [pimcore/pimcore#10911] [Q&A / Support] [answered]
Hi, 
I tried to upload an xml-file to create new DataObjects in my ProductClass. So I've chosen this resolver settings for the element creation etc.:
<img width="682" alt="Bildschirmfoto 2021-11-26 um 14 25 50" src="https://user-images.githubusercontent.com/68067510/143587966-c3173681-5792-47c9-9ea0-29091de575cd.png">
After the items where processed and imported, I wonder why the Name of the Objects are like this: 
<img width="347" alt="Bildschirmfoto 2021-11-26 um 14 21 26" src="https://user-images.githubusercontent.com/68067510/143588334-f34e6dcd-dfa8-4d10-8719-cb9d3964417f.png">

Is it possible to set the name of an object manually or in the XML-File somewhere instead of this "import-..."? 
Thanks in advance

**Comment by fashxp** [comment, marked as answer]
you can assign the key in the mappings configuration as any other data attribute... 
![image](https://user-images.githubusercontent.com/8792145/144389353-32ee945b-4ed4-404c-9025-3f5f82350196.png)

**Reply by MLindenhofer** [reply]
Thank you very much
